src/main/java/org/jitsi/videobridge/xmpp/MediaStreamTrackFactory.java

```diff
@@ -93,8 +94,11 @@ else if (SourceGroupPacketExtension.SEMANTICS_FID
                     List<SourcePacketExtension> simSources
                         = simGroup.getSources();
 
+                    int encodingslen = includeTemporal
+                        ? simSources.size() * 3 : simSources.size();
```

Comment:
worth defining a constant for the `3` here?  i assume it's something like "default num temporal layers"?